### ISSUE TITLE
Add event log mechanic

### ISSUE THREAD

**Issue by Nerigal** [LiskHQ/lisk#362]
For security purpose... 

it would be nice to add a view in an event.log or something...
To track "who does what with the API  identity base on publickey with the sort endpoint name call and the external ip 
for example :
`Mon Dec  19 10:51:17 2016 [44.55.66.77] GET /api/loader/status/sync 200 By=jb7yh4bcce531fjh86hs3fa8371c49b6898ce0895fe71jvg85jsec851c7mb86jf4
Mon Dec  19 10:51:18 2016 [44.55.66.77] GET /api/forging/status 403 By=jb7yh4bcce531fjh86hs3fa8371c49b6898ce0895fe71jvg85jsec851c7mb86jf4
Mon Dec  19 10:51:19 2016 [44.55.66.77] POST /api/forging/enable 403 By=jb7yh4bcce531fjh86hs3fa8371c49b6898ce0895fe71jvg85jsec851c7mb86jf4
`
Error code base on httpd RFC http://www.iana.org/assignments/http-status-codes/http-status-codes.xhtml
200 = success 
403 = unauthorized 

This could be very usefull to track and monitor activities on public nodes 
but also to secure private node users 

Thank you.

**Comment by 4miners** [comment]
I don't get that, publicKey is account identifier, it's not specified to node or request. Are you mean to add publicKeys for API whitelisting or just log publicKey only for forging-specyfic operations, for example: when someone trying to enable/disable forging and got rejected by whitelist?

**Comment by Nerigal** [comment]
In sort... You should be able to audit all api calls on your nodes...execute what, from where, by who 
the "who" is the publickey from the meaning public data "GET /api/accounts/getPublicKey?address=address" 
and so the endpoint ive wrote are just examples...

**Comment by 4miners** [comment]
That is already implemented:
```
[log] 2016-12-19 20:14:42 | POST /peer/blocks from 178.254.38.88
[log] 2016-12-19 20:14:42 | GET /peer/height from 5.9.70.174
[log] 2016-12-19 20:14:42 | GET /peer/height from 52.160.98.183
[log] 2016-12-19 20:14:42 | GET /peer/height from 158.69.221.136
[log] 2016-12-19 20:14:42 | GET /api/loader/status/sync from 127.0.0.1
[log] 2016-12-19 20:14:42 | GET /api/delegates/forging/status?publicKey=6a01c4b86f4519ec9fa5c3288ae20e2e7a58822ebe891fb81e839588b95b242a from 127.0.0.1
[log] 2016-12-19 20:14:42 | GET /peer/height from 149.154.65.119
[log] 2016-12-19 20:14:42 | POST /api/delegates/forging/enable from 127.0.0.1
```
There is still room to add response code and some additional details, however publicKey will not identify account that performed request in most cases.

**Comment by Nerigal** [comment]
i know about that...
but there is no response code 
The pubkey doesn't identify the source account from endpoint not requesting the pubkey but any call that as this data should be also recorded with the response code...

**Comment by karmacoma** [comment]
@Nerigal I agree, returning and logging appropriate status code is good thing. However, from what you have written it is not clear to me why you would want to log any `publicKey`. The `who` is answered by requesting the `ip` address.
